packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -458,7 +460,11 @@ export class API extends EventEmitter {
           try {
             var toAddress = B.Address.fromString(destinationAddress);
 
-            tx = new B.Transaction()
+            tx = (opts.coin === 'bch') ? new B.Transaction()
+              .from(utxos)
+              .to(toAddress, amount)
+              .fee(fee)
+              .sign(privateKey, B.crypto.Signature.SIGHASH_ALL | B.crypto.Signature.SIGHASH_FORKID, signingMethod) : new B.Transaction()
```

Comment:
what about this:
we keep txp version 3
we add `signingMethod` to TxProposal (defaults to schnorr on bch on new clients)
we add `signingMethod` parm to BTC's `.sign` and `.addSignatures` so we can remove all if